README.md

```diff
@@ -23,9 +23,32 @@ For more information on the operations supported by the `sql:Client`, which incl
 
 For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/mysql-init-options.html).
 
-
 ## Building from the Source
```

Comment:
```suggestion
## Building from the Source
```
We need to add a page TOC here.

Reply:
There is only 4 major headings,
Build from the source
Contributing to Ballerina
Code of Conduct
Useful Links

Is Table of Content necessary?

Reply:
Better to have one. However, if it is difficult to update now, that is fine.

Reply:
Ack. I'll add it in later PR